runner/src/server/services/uploadService.ts

```diff
@@ -224,4 +242,31 @@ export class UploadService {
     const promises = paths.map((path) => get<string>(path, {}));
     return Promise.all(promises);
   }
+
+  async uploadFilesS3(files) {
+    let response = new Array();
+
+    for (const file of files) {
+      const uploadParams = {
+        Bucket: bucketName,
```

Comment:
Can we add metadata here e.g. https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/s3-example-upload-bucket-item-with-metadata.html to identify which question a file is associated with? (This will help us identify it in the assessment tool later on)

Reply:
added some meta data